guacamole/src/main/webapp/app/form/directives/guacInputColor.js

```diff
@@ -0,0 +1,204 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* global Pickr */
```

Comment:
What is this?

Reply:
It's an annotation to advise the IDE that `Pickr` is a legitimate global and not something to warn about.

Reply:
Do you think that's worth documenting?

Reply:
The annotation or that `Pickr` is coming from an external library?

The `/* global ... */` syntax is relatively standard, recognized by jslint and eslint:

* https://www.jslint.com/help.html#global
* https://eslint.org/docs/rules/no-undef#rule-details

Reply:
Both I guess? Just mostly a comment about why that line is there, since it wasn't super obvious (to me anyway).

Reply:
Eh ... I'll just remove the annotation. If we move to using eslint or similar during build or CI at some point, we can then address if/how such annotations are added.